### PR TITLE
reverse #15734 partial and comment

### PR DESCRIPTION
**Pull request by 6543**
as title

ref: https://github.com/go-gitea/gitea/pull/15734#issuecomment-832974771

### CODE REVIEW

File: templates/repo/graph/commits.tmpl [hunk 1]
```diff
@@ -34,7 +34,8 @@
 							{{$refGroup := .RefGroup}}
 							{{if eq $refGroup "pull"}}
 								{{if or (not $.HidePRRefs) (containGeneric $.SelectedBranches .Name)}}
-									<a class="ui labelled icon button basic tiny" href="{{$.RepoLink}}/pulls/{{.ShortName|PathEscape}}">
+									<!-- it's intended to use issues not pulls, it it's a pull you will get redirected -->
```
Comment (6543) [inline templates/repo/graph/commits.tmpl]:
```suggestion
									<!-- it's intended to use issues not pulls, if it's a pull you will get redirected -->
```

Comment (6543) [reply templates/repo/graph/commits.tmpl]: if you have better comment, just suggest :)

**Review by zeripath — APPROVED** [review]
I'm sure that this should have already worked for #15730

**Comment by 6543** [comment]
ref #15730

File: templates/repo/graph/commits.tmpl (hunk 1 above)
Comment (wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf) [reply templates/repo/graph/commits.tmpl]: the first part of the sentence could just state that "use of 'issues', not 'pulls' here is intended" and perhaps links to this thread could be added, but I's fine.

**Review by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf — APPROVED** [review]
gimme back my redirects

**Review by KN4CK3R — APPROVED** [review]

**Pull request merged**